testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -11,25 +11,24 @@
 # * if there is no private network ($private_net is nil)
 # * if there is no PXE boot minion ($pxeboot_mac is nil)
 
+@buildhost
```

Comment:
Do we need @buildhost tag here?

Reply:
Yes. You can't run the retail tests if you don't have an OS image, and the OS image needs the build host to be built.